stdlib/jwt/src/main/ballerina/src/jwt/outbound_jwt_auth_provider.bal

```diff
@@ -59,19 +62,22 @@ public type OutboundJwtAuthProvider object {
         return authToken;
     }
 
-    # Inspect the incoming data and generate the token for JWT authentication.
-    #
-    # + data - Map of data which is extracted from the HTTP response
-    # + return - String token, or `auth:Error` occurred when generating token or `()` if nothing to be returned
+# Inspects the incoming data and generates the token for JWT authentication.
+#```ballerina
+# string|auth:Error? result = inspect(data);
+# ```
```

Comment:
Let's remove this code snippet here. This function was added in order to maintain object-equivalency and it does nothing other than returning `()`
```suggestion
```